package.json

```diff
@@ -6,12 +6,15 @@
   "scripts": {
     "lint": "node_modules/.bin/eslint ./",
     "server": "node_modules/.bin/http-server",
+    "pretest": "npm run server:json",
+    "posttest": "kill $(lsof -t -i:5000)",
```

Comment:
Regarding a great developer experience - should we support development on windows? Many libraries run Windows desktops for their staff.